src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/BeneficialOwnerIndividualDto.java

```diff
@@ -24,6 +25,10 @@ public class BeneficialOwnerIndividualDto {
     public static final String BENEFICIAL_OWNER_NATURE_OF_CONTROL_TYPES_FIELD = "beneficial_owner_nature_of_control_types";
     public static final String TRUSTEES_NATURE_OF_CONTROL_TYPES_FIELD = "trustees_nature_of_control_types";
     public static final String NON_LEGAL_FIRM_MEMBERS_NATURE_OF_CONTROL_TYPES_FIELD = "non_legal_firm_members_nature_of_control_types";
+    public static final String NON_LEGAL_FIRM_CONTROL_NATURE_OF_CONTROL_TYPES_FIELD = "non_legal_firm_control_nature_of_control_types";
+    public static final String TRUST_NATURE_OF_CONTROL_TYPES_FIELD = "trust_control_nature_of_control_types";
```

Comment:
To keep the names consistent, it might be preferable to call it TRUSTEES_CONTROL_NATURE_OF_CONTROL_TYPES_FIELD = trustees_control_nature_of_control_types

Reply:
But this relates to a trust NOT a trustee so I think it's correct. If it is wrong, web code would have to change too, as it defines `trust_control_nature_of_control_types`.

Reply:
The one at line 26 is trustee not trust
And it is control over trustee not trust, look at the ticket https://companieshouse.atlassian.net/browse/UAR-1595

Reply:
I think the names have been decided upon based on the NOC headings. Heading for this one is "The beneficial owner has the right to exercise, or actually exercises, significant control or influence over the activities of a trust" (which doesn't mention 'trustee'). The heading related to line 26 is "The beneficial owner is a trustee of a trust and" so guessing why that (long ago) was named TRUSTEES_NATURE_OF_CONTROL_TYPES_FIELD.

Personally, I'm happy with this naming. Any other opinions?

Reply:
Mapping will change in chips-filing-consumer